invenio_app_ils/vocabularies/cli.py

```diff
@@ -56,6 +56,39 @@ def index_json(filenames, force):
     click.echo("indexed {} vocabularies".format(index_count))
 
 
+@index.command(name="languages")
+@click.option("--force", is_flag=True)
+@with_appcontext
+def index_languages(force):
```

Comment:
could you add this command to our setup environment recipes? or on cds-ils to `fixtures vocabularies`, so that we will not forget to initialize the languages.

Reply:
I do that here: 
https://github.com/inveniosoftware/invenio-app-ils/blob/1059d00476a113fd819dd615c0efa50e9abb9dc2/invenio_app_ils/cli.py#L1637
and here:
https://github.com/CERNDocumentServer/cds-ils/blob/6599957c20b2b37e3997bdad98100b0a6fd709f6/cds_ils/cli.py#L242